OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDashboardFragment.java

```diff
@@ -219,6 +216,46 @@ public void onMenuItemClick(@NonNull ComponentName componentName, @NonNull Share
                         intent.setComponent(componentName);
                         startActivity(intent);
                     }
+
+                    @NonNull
+                    private String getFacebookSharingText() {
+                        String shareText;
+                        final String utmParams = courseData.getCourse().getCourseSharingUtmParams("facebook");
+                        String courseUrl = courseData.getCourse().getCourse_about();
+                        if (!TextUtils.isEmpty(utmParams)) {
+                            courseUrl += "?" + utmParams;
+                        }
+                        shareText = ResourceUtil.getFormattedString(
+                                getResources(),
+                                R.string.share_course_message,
+                                "platform_name",
+                                getString(R.string.platform_name)).toString() + "\n" + courseUrl;
+                        return shareText;
+                    }
+
+                    @NonNull
+                    private String getTwitterSharingText(@Nullable String twitterTag) {
```

Comment:
After we have standardised the utm param keys in `ShareUtils.ShareType` enum, we can add a param to this function of the same type and use it for facebook's case as well.

Consequently removing the `getFacebookSharingText` function and renaming this function to maybe `getSharingText` ?